recipes/statista/LICENSE.md

```diff
@@ -0,0 +1,675 @@
+### GNU GENERAL PUBLIC LICENSE
```

Comment:
i don't think you need this file do you? isn't it included in your github archive?